include/solvers/heat_transfer_scratch_data.h

```diff
@@ -427,9 +424,9 @@ class HeatTransferScratchData
    */
   FEValuesExtractors::Vector velocities;
   // This FEValues must mandatorily be instantiated for the velocity
-  FEValues<dim>               fe_values_navier_stokes;
-  std::vector<Tensor<1, dim>> velocity_values;
-  std::vector<Tensor<2, dim>> velocity_gradient_values;
+  FEValues<dim>               fe_values_fd;
+  std::vector<Tensor<1, dim>> velocity_values_fd;
```

Comment:
I think that since the velocity_values vector of tensor is independant of the number of fluids, maybe we should keep it as velocity_values